packages/wheelhouse/src/developmentActions.js

```diff
@@ -47,3 +47,23 @@ export const developmentLog = (pkgName, text) => dispatch => {
 export const developmentBuild = () => async (dispatch, getState) => {
   await dispatch(configLoad());
 };
+
+export const developmentLint = () => async (dispatch, getState) => {
+  await dispatch(configLoad());
+
+  const { packages } = getState();
+  let errored = "";
```

Comment:
this is the newer version. the app names are separated by a space and are preceded by the specific error message. i'm not crazy about the nested for loop, so maybe we just throw the error and don't specify the apps? 

  const { packages } = getState();
  let errored = "";
  for (let key in packages) {
    for (let lintingKey in packages[key].linting) {
      if (packages[key].linting[lintingKey]) {
        if (lintingKey === "chartAndPackageMatch" && !errored.length) {
          errored += "package.json and Chart.yaml are not in sync in:";
        }
        if (lintingKey === "chartAndPackageMatch") {
          errored += ` ${key}`;
        }
      }
    }
  }

  if (errored.length > 0) {
    throw new Error(`${errored}`);
  } else {
    log("👍");
  }
};


if i go with an array, do i loop through and throw a new error for each app or just concatenate into another string?